E45F01FFFE38DA63

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "E45F01FFFE38DA63",
+    "servers": [
+      {
+        "server_address": "eu1.cloud.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 44,
+    "ref_longitude": 10,
```

Comment:
Again, please use proper lat/long, this is too coarse.
Also, is the EUI really coming from the mac address of your network interface or did you make it up?

Reply:
i've update the lat and the long
![eui](https://user-images.githubusercontent.com/61233221/128678804-a7cf9a68-03a6-47c0-b9c8-e17bd2824c67.png)
The EUI it coming from here